projects/optic/src/commands/diff/upload-diff.ts

```diff
@@ -48,6 +48,13 @@ export async function uploadDiff(
       const currentBranch = await Git.getCurrentBranchName();
       if (currentBranch !== 'HEAD') {
         tags.push(sanitizeGitTag(`gitbranch:${currentBranch}`));
+      } else {
+        logger.warn(
```

Comment:
Would be good to only show this warning if there's not a manually set tag from the CLI.